src/navigation/checkout/Summary.js

```diff
@@ -164,6 +165,7 @@ class Summary extends Component {
     return (
       <FlatList
         data={ this.props.cart.items }
+        alwaysBounceVertical={false}
```

Comment:
Should this change only apply for iOS?
If yes, you can use conditionals `alwaysBounceVertical={ Platform.OS === 'ios' ? false : true }`